assets/flox.zdotdir/.zlogin

```diff
@@ -0,0 +1,10 @@
+if [ -f /etc/zlogin ]
```

Comment:
```suggestion
# Source /etc/zlogin and "${FLOX_ORIG_ZDOTDIR:-$HOME}/.zlogin" if they exist
#
# This script are called through `ZDOTDIR=<.../flox.zdotdir> zsh --no-globalrcs`,
# thus flox has to source the appropriate rc files itself.
# The value of $ZDOTDIR set before calling `flox activate`
# is preserved as `$FLOX_ORIG_ZDOTDIR`


if [ -f /etc/zlogin ]
```

Reply:
Done in 2c86b1a88e853180afead2a3d7c87541f743cb3c